apps/ledger-live-mobile/src/components/ChangeDeviceLanguageAction.tsx

```diff
@@ -15,6 +15,13 @@ type Props = {
   onStart?: () => void;
   onResult?: () => void;
   onError?: (err: Error) => void;
+  /*
+   * Defines in what type of component this action will be rendered.
+   *
+   * If "drawer", the component will be rendered as a content to be rendered in a drawer.
+   * If "view", the component will be rendered as a view. Defaults to "view".
+   */
+  renderedInType?: "drawer" | "view";
```

Comment:
I feel like if this is only needed by the `BluetoothDisabled` component we could avoid to add this prop that has to be passed down everywhere (prop drilling) and instead use a simple context:
- `RenderedInTypeContext = React.createContext({renderedInType: "view"})`
- default value of the context is "view"
- wrap content of drawers (all except one in LLM are implemented with `QueuedDrawer` I think so just 1 place to change) with <RenderedInTypeContext.Provider value={{renderedInType: "drawer"}) />
- use in `BluetoothDisabled`: `const {renderedInType} = useContext(RenderedInTypeContext)`

Reply:
also maybe instead of "view" use "fullScreen" or something less generic

Reply:
or maybe better with just a boolean like `isInDrawer: boolean` 🤷‍♂️

Reply:
That could be nice to use a context yeah